package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@aws/mynah-ui",
   "displayName": "AWS Mynah UI",
-  "version": "4.6.1",
+  "version": "5.0.0",
```

Comment:
Actually this is the first time we're doing a major and breaking change after mynah-ui went public. We're using release infos in the releases section. Each release contains detailed information about the change. But it will be better to introduce a deprecation warning and keep the old method until the deprecation target version is reached. 

Thanks for the flag!